controllers/entriesController.js

```diff
@@ -0,0 +1,44 @@
+const JournalEntry = require('../models/JournalEntry');
+
+const addEntry = (req, res) => {
+  const {
+    userId, title, entryBody, isPrivate,
```

Comment:
Did you leave out expectation of `moods` in request body because model wasn't available yet?